src/main/scala/beam/agentsim/infrastructure/ParkingManagerBenchmark.scala

```diff
@@ -95,28 +101,37 @@ object ParkingManagerBenchmark extends StrictLogging {
 
   val seed: Int = 42
 
-  val nTimes: Int = 5
+  val nTimes: Int = 1
```

Comment:
Should this be configurable?

Reply:
This will increase code complexity without giving much advantage to the user of this benchmark. But if we want to use it on a regular basis then I agree that we need to make all the params configurable here.

Reply:
Something to think about in the future - as I think I have seen that changed in other PRs - so it gets moved around I think.